pmd-plsql/src/test/resources/net/sourceforge/pmd/lang/plsql/ast/CreateTable.pls

```diff
@@ -4,8 +4,302 @@
 
 -- this is the customers.sql file:
 CREATE TABLE customers
-( customer_id number(10) NOT NULL,
-customer_name varchar2(50) NOT NULL,
-city varchar2(50),
-CONSTRAINT customers_pk PRIMARY KEY (customer_id)
-);
+    ( customer_id       number(10) primary key enable
+    , customer_name     varchar2(50) NOT NULL
+    , zip               NUMBER DEFAULT 10001 NOT NULL
+    , city              varchar2(50) default on null 'New York'
+    , status            VARCHAR2(1) CONSTRAINT status_chk CHECK (status in ('X', 'Y', 'Z'))
```

Comment:
We probably should extend the unit test to verify that the 5th column in the first create table statement has a inline constraint with type "check":

```java
        ASTTableColumn columnStatus = input.findChildrenOfType(ASTTable.class).get(0).findChildrenOfType(ASTTableColumn.class).get(4);
        Assert.assertEquals("status", columnStatus.getFirstChildOfType(ASTID.class).getImage());
        Assert.assertEquals(ConstraintType.CHECK, columnStatus.getFirstChildOfType(ASTInlineConstraint.class).getType());
```